pyOCD/target/__init__.py

```diff
@@ -78,7 +78,7 @@
           'kv11z7': target_MKV11Z128xxx7.KV11Z7,
           'kw01z4': target_MKW01Z128xxx4.KW01Z4,
           'kw40z4': target_MKW40Z160xxx4.KW40Z4,
-          'kw41z4': target_MKW41Z512xxx4.KW41Z4,
+          'kw41z512xxx4': target_MKW41Z512xxx4.KW41Z4,
```

Comment:
The name should be kept the same for now. If there is a switch to a new naming scheme, all the devices will be updated at the same time.